service-worker.js

```diff
@@ -70,7 +70,8 @@ self.addEventListener('message', function (event) {
 });
 
 // Pattern for ZIM file namespace - see https://wiki.openzim.org/wiki/ZIM_file_format#Namespaces
-var regexpZIMUrlWithNamespace = new RegExp(/(?:^|\/)([-ABIJMUVWX])\/(.+)/);
+// The ZIM file name (used as a prefix in the URL) is ignored
+var regexpZIMUrlWithNamespace = new RegExp(/(?:^|\/)([^\/]+\/)([-ABIJMUVWX])\/(.+)/);
```

Comment:
There's no real advantage of using the new RegExp constructor function here. You could just use:

`var regexpZIMUrlWithNamespace = /(?:^|\/)([^\/]+\/)([-ABIJMUVWX])\/(.+)/;`

But it's a matter of style, not substance.

Reply:
You're right, I'll change that.